language/de/common.php

```diff
@@ -685,6 +685,10 @@
 	'RETURN_TOPIC'				=> '%sZurück zum zuletzt besuchten Thema%s',
 	'RETURN_TO'					=> 'Zurück zu „%s“',
 	'RETURN_TO_INDEX'			=> 'Zurück zur Foren-Übersicht',
+
+	'ROUTE_NOT_FOUND'				=> 'Die angeforderte Route „%s” konnte nicht gefunden werden.',
+	'ROUTE_INVALID_MISSING_PARAMS'	=> 'An die Route „%s“ wurden ungültige oder fehlende Parameter übergeben.',
```

Comment:
Fehlende Parameter kann man nicht wirklich übergeben... Vielleicht besser:
```suggestion
	'ROUTE_INVALID_MISSING_PARAMS'	=> 'Parameter für die Route „%s“ sind ungültig oder fehlen.',
```